src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -36,7 +36,7 @@ const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? anonColorCondition : COLORS.white
+            backgroundColor: isSelect ? anonColorCondition : COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the card when it is not selected might affect readability if the text color inside the card is not adjusted accordingly. Please ensure that the text color contrasts well with the new background color for better visibility and user experience.
```diff
- backgroundColor: isSelect ? anonColorCondition : COLORS.white
+ backgroundColor: isSelect ? anonColorCondition : COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->